### PR TITLE
feat(website): create a basic blog on the doc site

### PR DESCRIPTION
**Pull request by SiTaggart**
Create the most basic of basic blogs on the site for us to publish articles on

### CODE REVIEW

**Comment by katieporter** [comment]
I love that this exists 😍

Some design nits:

On the index page -
- Could the top and bottom margin around the avatar/name/date be equal? Maybe space-40 or so?
- Could the date format be Month D, Yr or D Month, Yr? No preference, just looks a little more polished to have the full name of the month and drop the hyphenation.
- Can we make it such that every post has a preview snippet? Aayush's article looks funny without one.
- Not sure if this is okay...but could the article titles be black instead of blue? Still maintaining the same hover interactions. They're just like, really blue :hahaa: and I feel like it's obvious that they are clickable, even if they're black.

On the article page -
- Can the avatar/name/date stay the same size as they were on the index page? They feel a little chunkaay.
- Could we drop the top image from Shadi's article? It'll be cool to bring some illos in when we do a design pass to add a hero image and images to the index page, but just sitting at the top of the article, it feels a little out of place.
- It feels funky that Shadi's post as an in-page nav, but Aayush's post doesn't...I don't have a suggestion for how to fix that because I don't think the little chat bubbles could move inline...maybe we get rid of the in-page nav on Shadi's?
- Could we have more space between the article content and the chat bubbles on Aayush's post? space-140?
- Can the body of the articles be the same width? Also, can they be a little less wide? size-70?

File: packages/paste-website/src/pages/articles/2021-04-29-insights-&-metrics-that-inform-the-paste-design-system.mdx [hunk 1]
```diff
@@ -0,0 +1,288 @@
...
+      }
+    }
+  }
+`;
+
+<ArticleHeader
+  title={props.data.mdx.frontmatter.title}
+  description={props.data.mdx.frontmatter.description}
+  date={props.data.mdx.frontmatter.date}
+  author={props.data.mdx.frontmatter.author}
```
Comment (TheSisb) [inline packages/paste-website/src/pages/articles/2021-04-29-insights-&-metrics-that-inform-the-paste-design-system.mdx:78]: Would be cool to add the Twitter account in the remark metadata and link it in here behind author names

**Review by TheSisb — APPROVED** [review]
1000000000%

File: packages/paste-website/src/components/Article.tsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        <SiteLink to={article.frontmatter.slug}>{article.frontmatter.title}</SiteLink>
+      </Heading>
+      <Box marginBottom="space70" marginTop="space20">
+        <Stack orientation="horizontal" spacing="space40">
+          <Avatar name={article.frontmatter.author} src={article.frontmatter.avatar} size="sizeIcon60" />
+          <Box as="div">
+            <Text as="span" display="block" fontWeight="fontWeightBold" fontSize="fontSize30">
+              {article.frontmatter.author}
+            </Text>
+            <Text as="span" display="block" color="colorTextWeak" fontSize="fontSize20" lineHeight="lineHeight20">
```
[the remark this answers is not in the document]
Comment (SiTaggart) [reply packages/paste-website/src/components/Article.tsx]: Given it a go, can't be harmful

File: packages/paste-website/src/pages/articles/2021-04-29-insights-&-metrics-that-inform-the-paste-design-system.mdx (hunk 1 above)
Comment (SiTaggart) [reply packages/paste-website/src/pages/articles/2021-04-29-insights-&-metrics-that-inform-the-paste-design-system.mdx:82]: added `author_link`, just incase it's a portfolio site or something

**Comment by aayushpi** [comment]
```Growing Pains and How We Scaled our Design System Support
AI
Aayush Iyer
26 November, 20``` 

While linking external articles, cite with source and add language around "Posted with Permission"
<details><summary>Browser metadata</summary>





```
Path:      /articles/2020-11-26-growing-pains-and-how-we-scaled-our-design-system-support/
Browser:   Firefox 90.0 on Mac OS 10.15
Viewport:  1792 x 970 @1x
Language:  en-US
Cookies:   Enabled
```

[Open in BrowserStack](https://live.browserstack.com/dashboard#url=https%3A%2F%2Fdeploy-preview-1656--paste-docs.netlify.app%2Farticles%2F2020-11-26-growing-pains-and-how-we-scaled-our-design-system-support%2F&os=OS+X&os_version=Catalina&browser=Firefox&browser_version=90.0&resolution=1792x970&speed=1&start=true)



</details>

[Open Deploy Preview](https://deploy-preview-1656--paste-docs.netlify.app/articles/2020-11-26-growing-pains-and-how-we-scaled-our-design-system-support/) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=60e9017715294d00070b9168&commentID=60ec7f023d6d3f5271570040&resolution=resolved)

**Comment by aayushpi** [comment]
Title: Articles and Posts about Paste
<details><summary>Browser metadata</summary>





```
Path:      /articles/
Browser:   Firefox 90.0 on Mac OS 10.15
Viewport:  1041 x 970 @1x
Language:  en-US
Cookies:   Enabled
```

[Open in BrowserStack](https://live.browserstack.com/dashboard#url=https%3A%2F%2Fdeploy-preview-1656--paste-docs.netlify.app%2Farticles%2F&os=OS+X&os_version=Catalina&browser=Firefox&browser_version=90.0&resolution=1041x970&speed=1&start=true)



</details>

[Open Deploy Preview](https://deploy-preview-1656--paste-docs.netlify.app/articles/) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=60ecdc39e337c03c93131245&commentID=60ece40ba421f84f703acfeb&resolution=resolved)

**Review by nkrantz — APPROVED** [review]
Looking good, yay for articles

I think it makes sense to link out to Shadi's post on the Twilio blog (like you're doing with Aayush's), rather than duplicate it? Feels more consistent that way, and a little confusing to have "originally posted on twilio.com" link out to the original posting of it while Aayush's also links out but in a different way.

In that case the next blog post (stage categorization one) would be the first to actually be published on our blog.

One more note: should we call it something more fun than "articles"? Article hub? All Things Paste Blog? Something even better?

File: packages/paste-website/src/pages/articles/2021-04-29-insights-&-metrics-that-inform-the-paste-design-system.mdx [hunk 3]
```diff
@@ -0,0 +1,280 @@
+---
+title: Insights & Metrics that Inform the Paste Design System
+description: Learn how the Twilio Paste design system captures data to gain insights and track metrics which guide the development of the system.
+slug: /articles/2021-04-29-insights-&-metrics-that-inform-the-paste-design-system/
```
Comment (richbachman) [inline packages/paste-website/src/pages/articles/2021-04-29-insights-&-metrics-that-inform-the-paste-design-system.mdx:4]: Minor thing: should we encode the ampersand here, or make it "and"?

**Comment by SiTaggart** [comment]
@nkrantz 

> I think it makes sense to link out to Shadi's post on the Twilio blog (like you're doing with Aayush's), rather than duplicate it?

When we put your post on the site, we can change Shadi's back to a linked article. This gives us some prior art on how to do the next one easily. 

> should we call it something more fun than "articles"? Article hub? All Things Paste Blog? Something even better?

Went with Aayush's suggestion of "Articles and Posts about Paste" for now.

File: packages/paste-website/src/pages/articles/2021-04-29-insights-&-metrics-that-inform-the-paste-design-system.mdx (hunk 3 above)
Comment (SiTaggart) [reply packages/paste-website/src/pages/articles/2021-04-29-insights-&-metrics-that-inform-the-paste-design-system.mdx:4]: Probably easier to make it `and`

**Pull request merged**